.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Lint & Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m venv .venv
+          source .venv/bin/activate
+          (cd manage_arkime ; pip install -r requirements.txt)
+      - name: Lint with ruff
+        run: |
+          source .venv/bin/activate
+          # stop the build if there are Python syntax errors or undefined names
+          ruff --format=github --select=E9,F63,F7,F82 --target-version=py37 .
```

Comment:
How does the `--target-version` of Python 3.7 jive with the minimum Python version of 3.9 we have in our Python package (and above on line 15)?  Curious how we ended up with that specific choice.

Reply:
oops, i removed from the other ruff line but not that one, I'll do so now. I came up with 3.9 because 1 of our dependancies required 3.9. ;) 3.7 was there from the original copy from github.